Tools/freeze/freeze.py

```diff
@@ -124,7 +124,7 @@ def main():
 
     # default the exclude list for each platform
     if win: exclude = exclude + [
-        'dos', 'dospath', 'mac', 'macpath', 'macfs', 'MACFS', 'posix', ]
```

Comment:
'dospath'? Seriously? :wink:

Reply:
I don't know any of these modules (except of "posix"), so I chose to not touch this old code...